### PR TITLE
HDDS-6283. Change ContainerStateMachine ContainerOpExecutor name

### PR DESCRIPTION
**Pull request by guohao-rosicky**
## What changes were proposed in this pull request?

Change ContainerStateMachine ContainerOpExecutor name

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-6283

### CODE REVIEW

**Comment by guohao-rosicky** [comment]
@lokeshj1703 Please take a look.

**Review by lokeshj1703 — COMMENTED** [review]
@guohao-rosicky Thanks for working on this! The changes look good to me. NIT

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/ContainerStateMachine.java [hunk 1]
```diff
@@ -197,7 +198,11 @@ public ContainerStateMachine(RaftGroupId gid, ContainerDispatcher dispatcher,
     applyTransactionSemaphore = new Semaphore(maxPendingApplyTransactions);
     stateMachineHealthy = new AtomicBoolean(true);
 
-    this.executor = Executors.newFixedThreadPool(numContainerOpExecutors);
+    this.executor = Executors.newFixedThreadPool(numContainerOpExecutors,
+        new ThreadFactoryBuilder()
+            .setNameFormat("ContainerOP-" + gid.getUuid() + "-%d")
```
[the remark this answers is not in the document]
Comment (guohao-rosicky) [reply hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/ContainerStateMachine.java]: Has been modified, @lokeshj1703 Please take a look.

**Comment by guohao-rosicky** [comment]
Has been modified, @lokeshj1703 Please take a look. Thanks.

**Review by lokeshj1703 — APPROVED** [review]

**Comment by lokeshj1703** [comment]
Thanks @guohao-rosicky for the contribution! I have committed the PR to master branch.

**Pull request merged**